openwisp_controller/config/static/config/css/admin.css

```diff
@@ -265,6 +265,16 @@ div.tabs-divider {
   padding: 17em 0;
   font-size: 16px;
 }
+
+/*
+To prevent overflow of large system context
+variables text such as `zt_identity_secret`
+we should set a max-width with the word-wrap property
```

Comment:
I'd shorten this, eg:

Prevents the overflow of large text in system
defined variables such as `zt_identity_secret`